extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -457,6 +457,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     }
     const loaderContext = new GmailLoaderContext(this.factory, msgEl, attachmentsContainerInner);
     attachmentsContainerInner = $(attachmentsContainerInner);
+    attachmentsContainerInner.parent().find(this.sel.numberOfAttachments).hide();
```

Comment:
Attachments count label should be visible (as per https://github.com/FlowCrypt/flowcrypt-browser/issues/5460#issuecomment-1784910402)

Reply:
Got it - thank you! I understand better why the email `PDF attached` counts 2, because the public key is also part of the count in addition to the pdf attachment.

Reply:
We do have the code to show the attachment count label only when the numbers of rendered rendered attachments are more than 2 items. 

I guess this is all good just the count needs to ensure to be correct? It will be fixed then with the later changes.

Reply:
@sosnovsky I have fixed the over all issue for `noname` attachment, incorrect number of attachments, and showing the label for attachments. Only the Gmail test, not working quite well.